packages/stark-ui/src/modules/date-picker/components/date-picker.component.ts

```diff
@@ -92,48 +174,305 @@ export class StarkDatePickerComponent extends AbstractStarkUiComponent implement
 	@Input()
 	public pickerName: string = "";
 
+	/**
+	 * Placeholder to be displayed in the datepicker
+	 */
+	@Input()
+	public placeholder: string;
+
+	/**
+	 * If the dropdown is required or not. by default, the dropdown is not required
+	 */
+	@Input()
+	public required: boolean;
+
+	/**
+	 * Source Date to be bound to the datepicker model
+	 */
+	@Input()
+	public get value(): Date | null {
+		return this._value;
+	}
+
+	public set value(value: Date | null) {
+		if (!_isEqual(this._value, value)) {
+			// tslint:disable-next-line:no-null-keyword
+			this._value = value;
+			this.stateChanges.next();
+		}
+		this.cdRef.detectChanges();
+	}
+
+	/**
+	 * @ignore
+	 * @internal
+	 */
+	private _value: Date | null;
+
 	/**
 	 * Output that will emit a specific date whenever the selection has changed
 	 */
 	@Output()
-	public dateChanged: EventEmitter<Date> = new EventEmitter<Date>();
+	public dateChange: EventEmitter<Date | null> = new EventEmitter<Date | null>();
+
+	/**
+	 * Output that will emit a specific date whenever the input has changed
+	 */
+	@Output()
+	public dateInput: EventEmitter<Date | null> = new EventEmitter<Date | null>();
 
 	/**
 	 * Reference to the MatDatepicker embedded in this component
 	 */
 	@ViewChild(MatDatepicker)
-	public picker: MatDatepicker<any>;
+	public picker: MatDatepicker<moment.Moment>;
 
 	/**
 	 * Reference to the MatDatepickerInput embedded in this component
 	 */
 	@ViewChild(MatDatepickerInput)
-	public pickerInput: MatDatepickerInput<any>;
+	public pickerInput: MatDatepickerInput<moment.Moment>;
+
+	/**
+	 * @ignore
+	 * @internal
+	 */
+	public inputMaskEnabled: boolean = false;
+
+	/**
+	 * @ignore
+	 * @internal
+	 */
+	// tslint:disable-next-line:no-null-keyword
+	public ngControl: NgControl | null = null;
```

Comment:
For this, it's due to `MatFormFieldControl`... 
We need to implement `readonly ngControl: NgControl | null;`

But I agree with the fact I would prefer also to use `undefined` instead of `null` 😊